.github/integration/tests/sda/09_healthchecks.sh

```diff
@@ -22,7 +22,7 @@ if [ "$response" != "200" ]; then
 	exit 1
 fi
 
-response="$(curl -s -k -LI "http://s3inbox:8000/health" -o /dev/null -w "%{http_code}\n")"
+response="$(curl -s -k -L "http://s3inbox:8000/health" -o /dev/null -w "%{http_code}\n")"
```

Comment:
I'm curious why use GET instead of HEAD request given that what we want here is only the HTTP status code?
Also the HEAD request is used in the previous command.

Reply:
Good question, I see your point..! It's just to make sure that a GET request does work, in contrast to `curl -LI "http://s3inbox:8000" ` where you need to do a HEAD to get the health. Maybe both should be tested. I would argue that this is good enough for now, anyway :).